### ISSUE TITLE
(#202) Theme blog landing page

### ISSUE THREAD

**Issue by angievanhegan** [MantaRayMedia/alnap#39]
### Done criteria
Please add link to the blog page from each item in the view when theming
Card fields:
- [x] BLOCKER: AJAX to keep filters expanded after searching
- [x] Image (optional) 
- [x] BLOG [label on views] 
- [x] Date: 17 January 2017 
- [x] Title  
- [x] Author name 
- [x] Tags: ALNAP / MEMBER 
- [x] Switch between list/card display
- [x] Search filters
- [x] Sort filter
- [x] Download buttons
---
### Designs
- [Blog landing](https://invis.io/7CC18FLJ4)
- [Blog](https://invis.io/NJC8OGC2B)
- [Blog author](https://invis.io/SBCGDY838)
---
### ALNAP to check
[Blog landing page](http://alnap-dev.mrmdev.co.uk/blogs)
- [x] Blog landing page looks as expected
- [x] Buttons work
- [ ] Filters work
- [x] Views work
- [x] New blogs appear on this page

**Comment by angievanhegan** [comment]
@russ-manta needs assigning

**Comment by russ-manta** [comment]
@nadia847 there seems to be a random blog node or something displaying beneath the view - can this be removed? http://alnap-dev.mrmdev.co.uk/blogs

**Comment by nadia847** [comment]
@russ-manta removed

**Comment by angievanhegan** [comment]
@russ-manta Can this be moved to ALNAP review?

**Comment by Maria-alnap** [comment]
@russ-manta @nadia847 @angievanhegan I've checked most of the above criteria. The only one I haven't been able to check is the filters, as we haven't added any tags or authors to blogs yet.

**Comment by Maria-alnap** [comment]
@angievanhegan  @nadia847 one thing that needs to be edited here is the content in the first blue button "idea for a blog?". The i there needs to be capitalised

**Comment by angievanhegan** [comment]
@nadia847 please can you do this last small issue? Thanks

**Comment by angievanhegan** [comment]
@Maria-alnap  we can come back to the filter later on when the content is done.

**Comment by nadia847** [comment]
@Maria-alnap you can edit the links by editing the page. They are in a content block nugget in the Additional content section. Let me/matty know if you have any trouble doing this.

**Comment by Maria-alnap** [comment]
@nadia847 @matty-manta  apologies but when I open the content block nugget in either of the existing blog pages (not sure which one is the final?), all the cells show up empty (hence nothing to edit). Am I doing something wrong?

**Comment by Maria-alnap** [comment]
@matty-manta @nadia847 it would be good to also know how to tag blogs?

**Comment by matty-manta** [comment]
Hey @Maria-alnap are you looking within the Additional content tab rather than the Main content tab? There should be two nuggets, one content block nugget and one feed nugget. Take a look, if you still can't find it maybe it's worth us having a skype/slack call so you can share screens.

For tagging blogs, you just need to head to the "Related" tab - tags are in there. Just add a comma to separate individual tags. Once you type in the tags field, it will come up with suggestions based on the list of available tags, which you can manage here:

http://alnap-dev.mrmdev.co.uk/admin/structure/taxonomy/tags

Make sure you're selecting an existing tag - typos will create a new entry on that list.

Hope that all makes sense - let me know otherwise!

**Comment by angievanhegan** [comment]
@Maria-alnap This is ready for you.

**Comment by Maria-alnap** [comment]
@matty-manta I found it! I have edited it too. Only thing is that the buttons are showing up at the top of the page for some reason. That page might need a tweak from your end (see screenshot below)
![blog buttons.PNG](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/593fc96377a396001a3a210d/125516c66e82c728ace21e0d46ea989524c281bc9bfdb04db71fbcd6395428792c07b7692ac7b671ec132856595a4cf9405c0917fcf98b78f0e06e238c0f19f1c32f0e2a92624cf5f2d75dad0ba8218b4c8cb0b68b49c0ea3a24ee8d8c473f16000ac4.PNG)
